content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -188,12 +189,37 @@ For the node, pod, and service objects, the following network flows are supporte
 * Node -> Pod
 * Pod -> Node
 
+##### IPAM
 The following IPAM options are supported on Windows:
 
 * [Host-local](https://github.com/containernetworking/plugins/tree/master/plugins/ipam/host-local)
 * HNS IPAM (Inbox platform IPAM, this is a fallback when no IPAM is set)
 * [Azure-vnet-ipam](https://github.com/Azure/azure-container-networking/blob/master/docs/ipam.md) (for azure-cni only)
 
+##### Services & Load Balancing
+The following settings applicable to services and load balancing are available on Windows:
```

Comment:
nit:
```suggestion
##### Load balancing and Services

On Windows, you can use the following settings to configure Services and load balancing behavior:```

This wording makes it clear that “Service” refers to the Kubernetes resource by that name.